.circleci/config.yml

```diff
@@ -47,6 +47,9 @@ jobs:
       - run:
           name: Run Linter
           command: yarn lint
+      - run:
+          name: Update Past Orders tests
+          command: yarn test:update-past-orders
```

Comment:
Add new command as these tests failed every new day

Reply:
Maybe not the best long term solution. We can review this later